7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -0,0 +1,12 @@
+library(sf)
+library(ggplot2)
```

Comment:
I don't think we want these library calls here

Reply:
oh whoops, thanks - meant to move those